src/frontend/src/helpers/Storage/LocalStorage.ts

```diff
@@ -129,26 +139,40 @@ class LocalStorage implements AbstractStorage {
   }
 
   public async getFileToRun(proj: ProjectID, question: string): Promise<FileID|false> {
-    let p = await this.getProject(proj);
-    return p.runs[question] || false;
+    this.debug && console.log(`getFileToRun`);
```

Comment:
I understand your reasoning. But what if you want to run each conflict version and see which one runs..?